src/Microsoft.Azure.WebJobs.Host/Bindings/AttributeCloner.cs

```diff
@@ -292,10 +337,13 @@ private static string TemplateBind(IResolutionPolicy policy, PropertyInfo prop,
         {
             if (bindingData == null)
             {
+                // Skip validation if no binding data provided. We can't do the { } substitutions. 
```

Comment:
We can't validate without the binding data. But this is to support GetNameResolvedAttribute(), in which case we don't want validation anyways. 

---
In reply to: [123054807](https://github.com/Azure/azure-webjobs-sdk/pull/1188#discussion_r123054807) [](ancestors = 123054807)